src/DynamoDbCache.php

```diff
@@ -582,21 +593,29 @@ private function getKey(string $key): string
      */
     private function getRawItem(string $key): array
     {
-        $input = [
-            'Key' => [
-                $this->primaryField => [
-                    'S' => $key,
-                ],
-            ],
-            'TableName' => $this->tableName,
```

Comment:
I moved this out so that it would be clear it couldn't cause any exceptions.

Reply:
Makes sense.